app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -145,7 +145,13 @@ def references_path
     end
 
     def work_experience_path
-      if @application_form.application_work_experiences.any? || @application_form.work_history_explanation.present?
+      if @application_form.feature_restructured_work_history && FeatureFlag.active?(:restructured_work_history)
+        if @application_form.application_work_experiences.any? || @application_form.work_history_explanation.present?
+          Rails.application.routes.url_helpers.candidate_interface_restructured_work_history_path
+        else
+          Rails.application.routes.url_helpers.candidate_interface_restructured_work_history_review_path
```

Comment:
Should these branches be swapped? As in, if you've got any work experience inputted, go to the review. Otherwise, go  add some.

Reply:
Yes, you're correct. Changing now